core/scripts/server.js

```diff
@@ -144,20 +144,87 @@ app.get('*', (req, res, next) => {
       next()
     }
   }
-
+  const HTMLContent = `<html lang="en">
```

Comment:
i think this should be imported from separate file

Reply:
yeah, i had no idea it was included here btw, thought i emrged it before